lambdatimes/src/components/Content/Cards.js

```diff
@@ -2,10 +2,12 @@ import React, { Component } from 'react';
 import Card from './Card';
 
 const Cards = props => {
+  console.log(props);
   return (
     <div className="cards-container">
       {/* Using the cards prop, map over the list creating a 
-          new Card component for each passing the card as the only prop*/}
+          new Card component for each passing the card as the only prop*/ 
+          props.cards.map(card=><Card card={card} />)}
```

Comment:
Remember to always add a `key` prop when rendering a list of items.
```suggestion
          props.cards.map((card, index) => <Card card={card} key={index} />)}
```